thundra/constants.py

```diff
@@ -42,6 +42,14 @@
 CONTEXT_LOG_GROUP_NAME = 'log_group_name'
 CONTEXT_LOG_STREAM_NAME = 'log_stream_name'
 
+APPLICATION_ID_PROP_NAME = 'thundra.agent.application.id'
```

Comment:
You are getting them as environment variable but `.` can't be used in the environment variable of AWS Lambda function. This should not work. Have you tested this on AWS Lambda?

Reply:
Also, there must be `lambda` after `agent` in the prop name such as `thundra_agent_lambda_application_domainName`